.babelrc

```diff
@@ -12,6 +12,12 @@
     "@babel/preset-react"
   ],
   "plugins": [
+    [
+      "babel-plugin-styled-components",
+      {
+        "pure": true
```

Comment:
What is the effect of this ?

Reply:
Help to debug. Styled Component create classes with component names instead of only encryption.

`typography__StyledText-sc-19f9nb1-0 display1__TextDisplay1-sc-1dbfelc-0 successModal__SuccessTitle-sc-17o2h4e-2 kjwuIY`